larg4/pluginActions/ParticleListAction_service.cxx

```diff
@@ -114,16 +146,20 @@ namespace larg4 {
     // of the first EM particle that led to this one
     std::map<int,int>::const_iterator itr = fParentIDMap.find(trackid);
     while( itr != fParentIDMap.end() ){
-      mf::LogDebug("ParticleListActionService::GetParentage")
-      << "parentage for " << trackid
-      << " " << (*itr).second;
+      /*
+       * mf::LogDebug("ParticleListActionService::GetParentage")
+       * << "parentage for " << trackid
+       * << " " << (*itr).second;
+       */
```

Comment:
Suggest removing the code entirely instead of commenting it out.  Leaving it in as a comment causes maintenance issues.  We can always restore the code using the Git history.